controllers/admin/AdminStoresController.php

```diff
@@ -599,6 +599,6 @@ protected function adaptHoursFormat($value)
     {
         $separator = array_fill(0, count($value), ' | ');
 
-        return array_map('implode', $value, $separator);
+        return array_map('implode', $separator, $value);
```

Comment:
As of php 7.4, in the implode method, the separator must come before the array:

https://www.php.net/manual/en/function.implode.php